src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -115,6 +116,19 @@ export const ProgressReportApplication = ({ dar, datasets, readOnlyMode = true,
         onFormChange({ selectedDatasets: newDatasets, datasetIds: newDatasetIds });
     }
 
+    // Check if the DAR is a closeout review
+    // TODO: modify this logic for DAC chair when backend supports it
+    const isCloseoutReview = () => {
+        const user = Storage.getCurrentUser();
+        const isSigningOfficial = user.roles.some((role: UserRole) => role.name === 'SigningOfficial');
+        const isDacChair = user.roles.some((role: UserRole) => role.name === 'Chairperson');
```

Comment:
Note that this should return an instance of `DuosUser` which has all the user roles pre-calculated. For example, this is what I see in local storage for `Storage.getCurrentUser()`

![Screenshot 2025-06-13 at 7 50 10 AM](https://github.com/user-attachments/assets/f5582946-00c9-4624-9580-11379ffbf352)